tests/page-objects/forms/generic-form.po.js

```diff
@@ -64,6 +86,12 @@ module.exports = {
     checkBtn.click();
   },
 
+  reportApproveNative: async () => {
```

Comment:
I don't think this is doing what it's supposed to be doing anymore: namely to open the report review dropup. And hence, you had to add:
```
const reviewButton = element.all(by.css('.mm-icon-inverse')).get(0);
...
```
in those functions. 

I think it's better to move those here and leave the `invalidateReportNative` and `validateReportNative` just click on the validation buttons. 

So, have this function open the review dropup like: 
```
const reviewButton = element(by.css('.actions .fa-check')).element(by.xpath('..'));
await helper.waitElementToBeClickable(reviewButton);
```

Reply:
You were right about the function. Refactored it but used a different selector that I is less flaky.